vNext/test/NonSiloTests/project.json

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "dependencies": {
+    "System.Management.Automation": "6.1.7601.17515",
```

Comment:
Yes it is needed for DeployTest,  it has dependency on System.Management.Automation